modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/resources/less/style.less

```diff
@@ -645,6 +645,42 @@ body {
   overflow: unset !important;
   pointer-events: auto;
 }
+
+//===========================
+// Styles for column dragging
+.DragHandle {
+  flex: 0 0 16px;
+  z-index: 2;
```

Comment:
I'm guessing `z-index` is the depth of something, but what is the `flex`?

Reply:
flex is a layout system in css 
https://www.w3schools.com/css/css3_flexbox.asp

We use it extensively so it's worth learning it. this is a good course:

https://scrimba.com/g/gflexbox